estate/__manifest__.py

```diff
@@ -5,6 +5,8 @@
     'application': True,
     'data': [
         'security/ir.model.access.csv',
+        'views/estate_property_views.xml',
+        'views/estate_menus.xml',
```

Comment:
Usually we order files alphabetically.
```suggestion
        'views/estate_menus.xml',
        'views/estate_property_views.xml',
```

Reply:
I have changed the order of the files as you suggested but I'm facing the following error while activating the app in Runbot:
`ValueError: External ID not found in the system: estate.estate_property_action`

That error is because we have dependency between the files as `estate_property_action` needs to be defined first before being used in `estate_menus.xml` and The data files are sequentially loaded following their order in the `__manifest__.py` file.
That's why we need to have files in that order so that the action is defined in `estate_property_views.xml` before being used in `estate_menus.xml`.

Reply:
Yes, I missed that, my bad.